<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactDockWidget</class>
 <widget class="QDockWidget" name="AddContactDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a contact</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Network</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="networkComboBox"/>
    </item>
    <item>
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>Cellphone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cellphoneLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="nickLabel">
      <property name="text">
       <string>Nickname</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nicknameLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="inviteMsgLabel">
      <property name="text">
       <string>Invite Message</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="inviteMsgLineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Group</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="groupComboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Alert Profile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="alertProfileComboBox">
      <item>
       <property name="text">
        <string>General</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Meeting</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Silent</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="buttonsWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
